docs/libcurl/opts/CURLOPT_SSLCERT_BLOB.md

```diff
@@ -78,7 +79,7 @@ int main(void)
 # AVAILABILITY
 
 Added in 7.71.0. This option is supported by the OpenSSL, Secure Transport,
-Schannel and mbedTLS (since 7.78.0) backends.
+Schannel, mbedTLS (since 7.78.0) and wolfSSL (since 8.9.0) backends.
```

Comment:
Please use the exact phrase `(added in [version])` for the version information, to allow our automation to work. It can selectively hide this information in the output. The mbedTLS details should be marked up like that as well, the existing way is just an omission.